### ISSUE TITLE
False positive for rule "no-unused-vars"

### ISSUE THREAD

**Issue by doberkofler** [eslint/eslint#1794]
The following code:
```
(function () {
	"use strict";

	var test = 1,
		ok;

	if (test > 0) {
		ok = test;
	}
})();
```
generates a false positive in eslint 0.14.0:
```
5:2 - ok is defined but never used
```

**Comment by michaelficarra** [comment]
"use" means a read. It is written to, but never read.

**Comment by doberkofler** [comment]
@michaelficarra You are absolutely correct! I missed this subtle difference and like the rule even better now. Thank you.
